contrib/k8s/README.md

```diff
@@ -1,41 +1,123 @@
-# dex
+# Deploying dex on Kubernetes
 
-## Getting Started
+This document will allow you to set up dex in your Kubernetes; the configuration files used are generally useful, but will need to be modified to meet the needs of your deployment. The places that are likely to need modification will be called out as often as possible in this document.
```

Comment:
Nit: "set up dex in your Kubernetes **cluster**; the **example** configuration files ~~used~~ are generally useful"